vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -233,8 +251,8 @@ func (e *CaminoStandardTxExecutor) AddValidatorTx(tx *txs.AddValidatorTx) error
 			tx.Outs,
 			e.Tx.Creds[:len(e.Tx.Creds)-1],
 			0,
-			e.Backend.Config.AddPrimaryNetworkValidatorFee,
-			e.Backend.Ctx.AVAXAssetID,
+			e.Backend.Config.AddPrimaryNetworkValidatorFee, // TODO@ use baseFee?
```

Comment:
We should discuss TX per TX if we want to use base-fee. Some TX could cost lot of CAM (e.g. CreateChain), and there are TX for subnets where subnet's should be able to control fees.
Instead changing basefee directly we discussed some time a go a fee multiplicator, which could also directly control all the fees mentioned in todo here.